tasks/install-on-Darwin.yml

```diff
@@ -13,13 +13,21 @@
     state: latest
     update_homebrew: yes
 
+- name: Configure Buildkite service
```

Comment:
Would you mind adding a `when` to this to make it conditional, and opt-in? Internally, we handle this inside a follow-on role for some internal reasons.